rust/routee-compass-core/src/algorithm/component/scc.rs

```diff
@@ -148,26 +148,50 @@ fn largest_strongly_connected_component(graph: &Graph) -> Result<Vec<VertexId>,
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::model::graph::graph_config::GraphConfig;
-    use std::path::PathBuf;
+    use crate::model::property::{edge::Edge, vertex::Vertex};
+    use std::collections::HashMap;
 
     fn build_mock_graph() -> Graph {
-        // A test graph with 2 strongly connected components
-        let test_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
-            .join("src")
-            .join("algorithm")
-            .join("component")
-            .join("test");
-        let edge_list_csv = test_path.join("edges.csv").to_str().unwrap().to_string();
-        let vertex_list_csv = test_path.join("vertices.csv").to_str().unwrap().to_string();
-        let graph_conf = GraphConfig {
-            edge_list_csv,
-            vertex_list_csv,
-            n_edges: None,
-            n_vertices: None,
-            verbose: false,
+        let vertices = vec![
```

Comment:
That was because I moved the graph builder into the scope of the routee-compass application to be consistent with our other builders.

Reply:
hmm. but, all of the "parts" of building a graph are in compass-core. how do we create an instance of Graph if we're depending on compass-core but not compass-app? say, for example, if the eventual compass-http doesn't depend on CompassApp? it might be better to leave all of that functionality in core and let the graph_builder code be a pass-through, so that we can share that logic across downstream modules.

Reply:
Yeah that's a good point. I'll move the building functionality back into the core crate and let the compass application graph builder be a light wrapper around that